config.rb.sample

```diff
@@ -38,6 +38,11 @@ $new_discovery_url="https://discovery.etcd.io/new?size=#{$num_instances}"
 # "core-01" through to "core-${num_instances}".
 #$instance_name_prefix="core"
 
+# Change the version of CoreOS to be installed
+# The default value is "current", which points to the current version
+# of the selected channel
```

Comment:
Please add something like this to the comment:

To deploy a specific version, simply set $image_version accordingly. For example, to deploy version 709.0.0, set $image_version="709.0.0".